src/file-utils/edfi_lms_file_utils/constants.py

```diff
@@ -1,18 +1,38 @@
-# SPDX-License-Identifier: Apache-2.0
-# Licensed to the Ed-Fi Alliance under one or more agreements.
-# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-# See the LICENSE and NOTICES files in the project root for more information.
-
-ASSIGNMENT = "assignment"
-ASSIGNMENTS = "assignments"
-ATTENDANCE_EVENTS = "attendance-events"
-GRADES = "grades"
-LMS_SECTION_SOURCE_SYSTEM_IDENTIFIER = "LMSSectionSourceSystemIdentifier"
-SECTION = "section"
-SECTIONS = "sections"
-SECTION_ASSOCIATIONS = "section-associations"
-SECTION_ACTIVITIES = "section-activities"
-SOURCE_SYSTEM_IDENTIFIER = "SourceSystemIdentifier"
-SUBMISSIONS = "submissions"
-SYSTEM_ACTIVITIES = "system-activities"
-USERS = "users"
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# Resources
+ASSIGNMENT = "assignment"
+ASSIGNMENTS = "assignments"
+ATTENDANCE_EVENTS = "attendance-events"
+GRADES = "grades"
+SECTION = "section"
+SECTIONS = "sections"
+SECTION_ASSOCIATIONS = "section-associations"
+SECTION_ACTIVITIES = "section-activities"
+SOURCE_SYSTEM_IDENTIFIER = "SourceSystemIdentifier"
+SUBMISSIONS = "submissions"
+SYSTEM_ACTIVITIES = "system-activities"
+USERS = "users"
+
+# Keys
+LMS_SECTION_SOURCE_SYSTEM_IDENTIFIER = "LMSSectionSourceSystemIdentifier"
+
+# Data types for dataframes
+SECTIONS_DATA_TYPES = {
+    "SISSectionIdentifier": "string",
+    "Title": "string",
+    "SectionDescription": "string",
+    "Term": "string",
+    "Status": "string",
+}
```

Comment:
Here's why I didn't place them here in the first place: these constants aren't intended for re-use in multiple places. Since they're only needed in the `file_reader.py` module, it is safe to leave them there.

If we have a change in the data types... this constants file might actually be easier to remember where to make the updates. When leaving constants inline (for example, in `file_reader`), there is sometimes the advantage that it is easier to understand the code while reading it because you can see the constant right there. In this case though, it won't make much of a difference I think. Someone reading `file_reader` does not need to know, immediately, about the details of the data types. So it is ok for the reader to just see `data_types = c.USERS_DATA_TYPES` and then they can quickly get to that constant definition if they actually care about the details.

 In general my habit is the original pattern - to extract constants to a separate file when they will be re-used. For this case, I think I'm convinced that it is actually a little better to make the change that you have done.

Reply:
What do you think about putting those constants at the beginning of the file_reader.py file? my plan is to reuse them in the same file but in new methods

Reply:
Top of the file or here in this constants class, either is fine with me.